kontrakter/siktelseTiltale/1.0/siktelseTiltale.schema.json

```diff
@@ -0,0 +1,870 @@
+{
```

Comment:
@calle-777 :
meldingsId:
Unik per sending eller per begjæring? (Altså vil denne beholdes ved korrigeringer?)
Bør dette være egen datatype? (Eks. GUID?)
Id'er.
Generelt - burde vært egen type for GUIDs. (med pattern)
Bør komme klart frem hvilke id'er som skal tilbake til politiet.
person:
internId - kunne vært local global id.
få det klarere hvordan de ulike id'ene er brukt - speskielt internId og unikId

Reply:
internId brukes på refferanser innenfor en melding og lever ikke utenfor en melding, hvis samme siktelse/begjæring sendes på nytt så vil de kanskje ha nye id'er.
Rydder i alle intern og unikId med bedre kommentarer hva de er. Standardiserer på bruk av GUID på interne id'er.